### DISCUSSION TITLE
Dash "-" in the project name is changed to underscore "_" when project is generated

### DISCUSSION THREAD

**Discussion by jakubgg** [TiagoSilvaPereira/vemto-issues#240] [Ideas & Suggestions]
**Describe the bug**
A name of the project containing dashes "-" (e.g. "sample-project") gets converted to a name with underscores "_" when creating a project directory (resulting in "sample_project"). 

**To Reproduce**
1. create "New Project"
2. name project "sample-project"
3. go to "Generate Project" click "Generate"
4. the project directory created has it's name changed to "sample_project"

**Expected behavior**
The name of the project and the name of the project directory should not be changed unless it contains characters formatted in a way that git disallows (or are illegal in a filesystem). 

**Desktop (please complete the following information):**
 - OS: MacOS 11.6
 - Version: 0.9.11

**Comment by TiagoSilvaPereira** [comment]
Hi @jakubgg thank you for your suggestion... I'm converting it into a discussion, as it is not a Bug, only a way I decided to generate the folder to facilitate the code generation process

**Comment by jakubgg** [comment]
Many thanks for opening this for a discussion @TiagoSilvaPereira ! 

My reasoning is that the change to underscores is a bit arbitrary, and the use of dashes is as common or even more common than underscores (if you look at your own git repos, you almost exclusively use dashes in names). 

It is a bit inconvenient to be forced to use the underscores, especially if you have different naming conventions in projects or would like to extend an already existing project.

Of course it all depends if this would require big changes to the code or is something easy to implement.
